### ISSUE TITLE
"editor.suggestSelection": "recentlyUsed" not working as expected

### ISSUE THREAD

**Issue by archcorsair** [Microsoft/vscode#43778]
### Issue Type
Bug

### Description

`"editor.suggestSelection": "recentlyUsed"`
I love this feature but I think its broken in this release. It is not remembing my recently used suggestion and only highlighting the top suggestion.

### VS Code Info

VS Code version: Code 1.20.1 (f88bbf9137d24d36d968ea6b2911786bfe103002, 2018-02-13T15:23:28.607Z)
OS version: Darwin x64 17.4.0

<details><summary>Extensions (19)</summary>

Extension|Author (truncated)|Version
---|---|---
EditorConfig|Edi|0.12.0
beautify|Hoo|1.3.0
html-css-class-completion|Zig|1.16.2
path-intellisense|chr|1.4.2
vscode-eslint|dba|1.4.5
githistory|don|0.4.0
vscode-babel-coloring|dza|0.0.4
gitlens|eam|8.0.0
vscode-npm-script|eg2|0.3.3
code-runner|for|0.8.7
easy-icons|jam|0.3.1
node-debug2|ms-|1.19.0
debugger-for-chrome|msj|4.1.0
vetur|oct|0.11.7
trailing-spaces|sha|0.2.11
code-spell-checker|str|1.6.3
vscode-react-native|vsm|0.5.7
vscode-todo-highlight|way|0.5.11
local-history|xyz|1.6.2

(1 theme extensions excluded)

</details>
Reproduces only with extensions

**Comment by egamma** [comment]
@archcorsair did you try the different setting values as described here? https://github.com/Microsoft/vscode/issues/41060#issuecomment-364284879?

**Comment by archcorsair** [comment]
@egamma I tried `byRecency` but it did not change anything. I'd also like to note that I didn't have any user settings configured for this feature up until I tried to troubleshoot why it stopped working.

First I tried manually setting `"editor.suggestSelection": "recentlyUsed"` according to the dropdown provided:
![image](https://user-images.githubusercontent.com/1036921/36335588-e61f1292-1335-11e8-9ff7-44eb8ffaac46.png)

I then tried updating it to `"editor.suggestSelection": "byRecency"` but it still does not suggest the most recently used item, only the top selection.
